### ISSUE TITLE
JSONLoader.load() behavior.

### ISSUE THREAD

**Issue by hrshdhgd** [mapping-commons/sssom-py#173]
When a JSON document is passed to`JSONLoader().load()` (from `linkml_runtime` package), it returns a `MappingSet` object as shown below:

***Input***
jsondoc = 
```
{
   "mapping_set_id":"http://w3id.org/sssom/mapping/tests/data/basic.tsv",
   "license":"https://creativecommons.org/publicdomain/zero/1.0/",
   "mappings":[
      {
         "subject_id":"x:appendage",
         "predicate_id":"owl:equivalentClass",
         "object_id":"y:appendage",
         "match_type":[
            "Lexical",
            "HumanCurated"
         ],
         "subject_label":"appendage",
         "subject_category":"biolink:AnatomicalEntity",
         "object_label":"appendages",
         "object_category":"biolink:AnatomicalEntity",
         "subject_source":"x",
         "object_source":"y",
         "mapping_tool":"rdf_matcher",
         "confidence":0.840714406,
         "comment":"."
      },
      {
         "subject_id":"x:appendage",
         "predicate_id":"owl:equivalentClass",
         "object_id":"z:appendage",
         "match_type":[
            "Lexical",
            "HumanCurated"
         ],
         "subject_label":"appendage",
         "subject_category":"biolink:AnatomicalEntity",
         "object_label":"APPENDAGE",
         "object_category":"biolink:AnatomicalEntity",
         "subject_source":"x",
         "object_source":"z",
         "mapping_tool":"rdf_matcher",
         "confidence":0.840714406,
         "comment":"."
      },
      {
         "subject_id":"x:bone_element",
         "predicate_id":"owl:equivalentClass",
         "object_id":"y:bone",
         "match_type":[
            "Lexical",
            "HumanCurated"
         ],
         "subject_label":"bone element",
         "subject_category":"biolink:AnatomicalEntity",
         "object_label":"bones",
         "object_category":"biolink:AnatomicalEntity",
         "subject_source":"x",
         "object_source":"y",
         "mapping_tool":"rdf_matcher",
         "confidence":0.534601961,
         "comment":"."
      }
   ],
   "creator_id":"cjm",
   "subject_source":"c",
   "object_source":"b",
   "mapping_tool":"https://github.com/cmungall/rdf_matcher",
   "mapping_date":"2020-05-30",
   "comment":"mock data"
}
```
***Output***
JSONLoader().load(source=jsondoc, target_class=MappingSet) = 
```

MappingSet(
    mapping_set_id='http://w3id.org/sssom/mapping/tests/data/basic.tsv', 
    license='https://creativecommons.org/publicdomain/zero/1.0/', 
    mappings=
    {
        'x:appendage': 
            Mapping(
                subject_id='x:appendage', 
                predicate_id='owl:equivalentClass', 
                object_id='z:appendage', 
                match_type=[
                    (text='Lexical', description='Lexical match'), 
                    (text='HumanCurated', description='Match based on human expert opinion')
                    ], 
                subject_label='appendage', 
                subject_category='biolink:AnatomicalEntity', 
                predicate_label=None, 
                object_label='APPENDAGE', 
                object_category='biolink:AnatomicalEntity', 
                author_id=[], 
                author_label=[], 
                reviewer_id=[], 
                reviewer_label=[], 
                creator_id=[], 
                creator_label=[], 
                license=None, 
                subject_source='x', 
                subject_source_version=None, 
                object_source='z', 
                object_source_version=None, 
                mapping_provider=None, 
                mapping_cardinality=None, 
                mapping_tool='rdf_matcher', 
                mapping_tool_version=None, 
                mapping_date=None, 
                confidence=0.840714406, 
                subject_match_field=[], 
                object_match_field=[], 
                match_string=[], 
                subject_preprocessing=[], 
                object_preprocessing=[], 
                match_term_type=None, 
                semantic_similarity_score=None, 
                semantic_similarity_measure=None, 
                see_also=[], 
                other=None, 
                comment='.'), 
        'x:bone_element': 
            Mapping(
                subject_id='x:bone_element', 
                predicate_id='owl:equivalentClass', 
                object_id='y:bone', 
                match_type=[
                    (text='Lexical', description='Lexical match'), 
                    (text='HumanCurated', description='Match based on human expert opinion')
                    ], 
                subject_label='bone element', 
                subject_category='biolink:AnatomicalEntity', 
                predicate_label=None, 
                object_label='bones', 
                object_category='biolink:AnatomicalEntity', 
                author_id=[], 
                author_label=[], 
                reviewer_id=[], 
                reviewer_label=[], 
                creator_id=[], 
                creator_label=[], 
                license=None, 
                subject_source='x', 
                subject_source_version=None, 
                object_source='y', 
                object_source_version=None, 
                mapping_provider=None, 
                mapping_cardinality=None, 
                mapping_tool='rdf_matcher', 
                mapping_tool_version=None, 
                mapping_date=None, 
                confidence=0.534601961, 
                subject_match_field=[], 
                object_match_field=[], 
                match_string=[], 
                subject_preprocessing=[], 
                object_preprocessing=[], 
                match_term_type=None, 
                semantic_similarity_score=None, 
                semantic_similarity_measure=None, 
                see_also=[], other=None, comment='.')
    }, 
        mapping_set_version=None, 
        mapping_set_source=[], 
        mapping_set_description=None, 
        creator_id=['cjm'], 
        creator_label=[], 
        subject_source='c', 
        subject_source_version=None, 
        object_source='b', 
        object_source_version=None, 
        mapping_provider=None, 
        mapping_tool='https://github.com/cmungall/rdf_matcher', 
        mapping_date='2020-05-30', 
        subject_match_field=[], 
        object_match_field=[], 
        subject_preprocessing=[], 
        object_preprocessing=[], 
        match_term_type=None, 
        see_also=[], 
        other=None, 
        comment='mock data'
        )

```

Is this the expected behavior @cmungall , @matentzn ?

**Comment by hrshdhgd** [comment]
For now I have implemented # 2 above

**Comment by matentzn** [comment]
I really dont get this at all.. Can you run it by Chris again?

**Comment by hrshdhgd** [comment]
fixed by #195
